Dockerfile

```
@@ -1,14 +1,21 @@
-# Build stage
+#BASE (OS)
 FROM node:20-alpine as build
-WORKDIR /app
-COPY package*.json ./
-RUN npm ci
+
+#WORKDIR
+WORKDIR app/
```

Hadolint (2.12.0)

[error] Use absolute WORKDIR

(DL3000)